.circleci/config.yml

```diff
@@ -320,7 +324,8 @@ jobs:
       - checkout-submodules
       - android/restore-gradle-cache
       - copy-gradle-properties
-      - update-gradle-memory
+      - update-gradle-memory:
+          jvmargs: "Xmx1024m"
```

Comment:
Could it be this new step that makes the Release Build job fail – as it updates the `gradle.properties` with a new value (1024 instead of the 2048 we used before) and later creates an error in the job when fastlane asks `"gradle.properties has changes that need to be merged. Would you like to see a diff?"` in interactive mode?

Reply:
Yep! It's surely that one :-) We fixed it for beta builds, but missed it for the release build.